sql/log.cc

```diff
@@ -626,8 +626,9 @@ bool write_bin_log_start_alter(THD *thd, bool& partial_alter,
     start_alter_info *info= thd->rgi_slave->sa_info;
     bool is_shutdown= false;
 
-    info->sa_seq_no= start_alter_id;
-    info->domain_id= thd->variables.gtid_domain_id;
+    info->gtid.domain_id= thd->variables.gtid_domain_id;
```

Comment:
Shall we make the assignment more compact as `info->gtid= { /* copy-list initialization */ ... }`

Reply:
Ooh, that wasn’t a thing in C. https://stackoverflow.com/a/18325112

Reply:
I noticed you also found useful [cppreference](https://en.cppreference.com/w/cpp/language/aggregate_initialization) which in this case spells out the copy-list assignment is good.